examples/lammps/Dockerfile

```diff
@@ -1,4 +1,5 @@
 # ch-test-scope: full
+# ch-test-builder-exclude: ch-grow  # See issue #890
```

Comment:
Is this going to be merged before #905 ?

Reply:
No; but we didn't write that down.